backend/lib/app.js

```diff
@@ -2,6 +2,22 @@
  * @param {import('fastify').FastifyInstance} fastify
  */
 export default async function (fastify) {
+  fastify.register(import('fastify-swagger'), {
+    routePrefix: '/docs',
+    openapi: {
+      info: {
+        title: 'CSAF Validator Service',
+        version: '1.0.0',
```

Comment:
Isn't that the version of the server? If so shouldn't that be retrieved from a Git tag?
(Maybe that's something for later - if so, please add as issue and ignore in the first round :-))

Reply:
Yes, this might be an option. I'll add this as an issue.

Reply:
Added as issue: https://github.com/secvisogram/csaf-validator-service/issues/5#issue-1196871889